### ISSUE TITLE
Nightly build failure on Cori due to SGType being undefined

### ISSUE THREAD

**Issue by ikalash** [gahansen/Albany#156]
My nightlies on the NERSC machine Cori began to fail last week due to an issue having to do with SGType not being defined: 

http://my.cdash.org/viewBuildError.php?buildid=1257570

This tests builds FELIX only.  @mperego, @bartgol : I know you were working on FELIX last week.  Is it possible if any of the changes you pushed caused this?  Or, perhaps develop branch of Trilinos is required now for FELIX to build?  On Cori, I build master, a difference b/w those nightly tests and the tests on my SRN Sandia workstation.

**Comment by bartgol** [comment]
Hi Irina,

The test fails because I changed the default behavior of ENABLE_STOKHOS: now it is off by default. Unfortunately, when ALBANY_FAST_FELIX is enabled, there is a need for SGType, which is only defined when ALBANY_STOKHOS is defined.

I changed ALBANY_STOKHOS to OFF by default since I thought it was more the "usual" case. Besides, not enabling things by default allows one to find bugs in the configuration (like this one). 

The quick fix for the nightly build is to add ENABLE_STOKHOS to the configure script. But a cleaner solution would be to enable stokhos in the CMakeLists.txt file whenevere ALBANY_FAST_FELIX is enabled. Furthermore, also ALBANY_SG and other configurations that require Stokhos should set ENABLE_STOKHOS to ON, to ensure the SGType is defined for them.

On a side node: ALBANY_FAST_FELIX is used (through an ifdef) only in Aeras. Should it perhaps change name into ALBANY_FAST_AERAS?
